modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -307,10 +307,26 @@ public void processOpts() {
                 }
             }
         }
+
+        processUsings(objs);
+
         // process enum in models
         return postProcessModelsEnum(objs);
     }
 
+    private void processUsings(Map<String, Object> objs) {
```

Comment:
this method is bound to cause lots of conflicts. For instance, add System.IO to imports and create a model called File.

I think the only solution to this concern is fully qualifying all usages of generated code.